mappings/b08255cc7747d22385b1346b5e6f9fad73dd7760e146f923d5caf1c30014df104e56.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "b08255cc7747d22385b1346b5e6f9fad73dd7760e146f923d5caf1c30014df104e56",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://neovikings.io",
```

Comment:
Hi @jshear,

on the website i only found policyId for the collection, but not for this asset. Could you please add policyId of this token to the website or twitter so we can verify the project?

Thanks!

Reply:
Hello @VladislavKudrin! The policy ID has been added to the footer of the website alongside the original NFT collection's policy ID. I'll make sure to have this done before making a PR for future registrations. Thank you!